### ISSUE TITLE
Suggestion: Dragoon feature to remove F&C/Wheeling Thrust from Chaos/Full Thrust Combos?

### ISSUE THREAD

**Issue by RandomSuggestions** [Grammernatzi/XIVComboPluginExpandedest#75]
I know the purpose of Combo Expandest is to remove the number of buttons you need to bind to your keyboard. However with Dragoons specifically having effectively two 5 button combos on a single key, it's this weird outlier among the melee DPS where those options feel too automated to have so many actions on a single button, and effectively impossible to map Dragoon to feel like my other melee jobs which enjoy a lot of Expandest options. 

With that in mind I would like to request two features; 

1.) Wheeling Thrust/Fang and Claw Toggle that de-activates the Fang and Claw/Wheeling Thrust actions off the Chaos and Full Thrust combos. 

2.) A Chaos Thrust/Full Thrust Combo Option in the same vein as Samurai's Gekko and Kasha Combo Options that will start those combo chains with Disembowel and Vorpal Thrust respectively. Besides looking much better than two repeating icons next to each other on my bars, it'll also allow me to play my Dragoon with similar muscle memory to the Expandest Samurai with those options enabled. 

Thank you for all the great work you're doing with this plugin!

**Comment by Grammernatzi** [comment]
There's already an option that makes Fang and Claw/Wheeling Thrust not automated and just splits them across the two combos. Have you tried this yet?

**Comment by RandomSuggestions** [comment]
So my immediate gut reaction here is that the Wheeling Thrust/Fang and Claw option that currently exists works is both adequate and feels _much_ better with the Chaos Thrust Disembowel option. It reads better and reducing the consecutive number of actions from five to three feels much better on the too-much-automation side of things.

As to whether or not we need a Vorpal Thrust combo, the best arguments I can make right now besides offering feature parity with Samurai (Since they both offer early combo branches) boils down to giving the user a choice between having a 2 or 3 button primary combo layout, and keeping the... I don't know what you'd call it, "Action Economy" even with three max consecutive actions on both the Disembowel and Vorpal Thrust buttons. 

TL;DR - The Wheeling Thrust/F&C Toggle doesn't feel anywhere near as necessary on the Chaos Thrust side of things, but I still think Vorpal Thrust would still be a positive addition. Worst case scenario if you don't add it officially is that I can fake it from here pretty comfortably.

**Comment by Grammernatzi** [comment]
Well, it's entirely a cosmetic thing, basically, but I might add it just because; who knows, maybe someone that wants True Thrust and Vorpal Thrust instead of True Thrust and Disembowel on their hotbar might like it. But, for now, it'll be later; apologies about that, but since it's entirely a visual thing, it is kind of a low priority.

**Comment by RandomSuggestions** [comment]
"Kind of a low priority."

(Implements it anyway 90 minutes later)

You fuckin' rock.

**Comment by Grammernatzi** [comment]
I needed some downtime anyway lmao, so I just figured, why not; was taking a break from learning Black Mage's level 90 meme rotation.
